<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>net for cifar10 by:demorz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="epoch">
        <property name="text">
         <string>epoch:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="train_acc">
        <property name="text">
         <string>train_acc:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="loss">
        <property name="text">
         <string>loss:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="test_acc">
        <property name="text">
         <string>test_acc:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="train_b">
        <property name="text">
         <string>开始训练</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quit_b">
        <property name="text">
         <string>结束训练</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="Lay557">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>538</width>
         <height>382</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="Lay"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="fold"/>
      </item>
      <item>
       <widget class="QPushButton" name="get_path">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="use_net">
        <property name="text">
         <string>分类文件夹中图片</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="cramera">
        <property name="text">
         <string>打开摄像头</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="anQ">
        <property name="text">
         <string>按Q退出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="test">
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
